### ISSUE TITLE
Can't launch multiple instances of the application that is using WebView2?

### ISSUE THREAD

**Issue by Merko** [MicrosoftEdge/WebView2Feedback#368]
So I've created a simple application using the WebView2 component.

But for some reason I can't launch the application twice. The second time the WebView2 control is just blank.

Am I doing something wrong, or is this intended? It works fine when I have the application  open in one instance, but when I run it again, the second instance is just blank.. 

Using: 0.9.579-prerelease
Thanks

[AB#28236607](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/28236607)

**Comment by douglas-jordan** [comment]
I have run many instances of the same app without issue.

**Comment by ukandrewc** [comment]
Probably need to see some code. Same as @douglas-jordan, I have tested a tabbed browser without any issues.

**Comment by Merko** [comment]
> Probably need to see some code. Same as @douglas-jordan, I have tested a tabbed browser without any issues.

That is weird. It's not really any code, just putting out the WebView2 component, navigating it to an URL, then open up the app again, but the WebView2 component isnt visible in the new window. Works fine if I exit the program, then open it up again, but cant have two windows of WebView2.. I don't understand why :) 

I also can't use Stable version of WebView2, only Pre-release (if I choose any of the other, it won't appear in the component list)

**Comment by pagoe-msft** [comment]
@Merko, 

Are you using .NET WebView2? WPF or WinForms? 

Could you send us sample code?

**Comment by ukandrewc** [comment]
> Are you using .NET WebView2? WPF or WinForms?
Winforms
I was just thinking of sorting out my tabbed sample and post it.
Will get round to it in the next couple of days.

**Comment by Merko** [comment]
> Are you using .NET WebView2? WPF or WinForms?
> 
> Could you send us sample code?

I'm using WinForms.
I recorded video of it here:

https://streamable.com/u5u29w

Code:
`    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load

        Dim URL As New Uri("https://bing.com")
        WebView.Source = URL

    End Sub`

Sample project:
https://edvin.no/WebView2Sample.zip
-

Again, maybe I'm doing it totally wrong, but I cannot figure out what and why :-)

**Comment by champnic** [comment]
I've got a repro and this looks like a bug on our side. Thanks for the details, we'll look into this!

**Comment by mikemeinz** [comment]
You are missing some essential lines of code.

```
 Sub WebViewTest_Load(sender As Object, e As EventArgs) Handles Me.Load
        AddHandler WebView.CoreWebView2.CoreWebView2Ready, AddressOf WV_CoreWebView2Ready
        InitializeAsync()
    End Sub

    Async Sub InitializeAsync()
        Await WebView.EnsureCoreWebView2Async()
    End Sub

    Private Sub WV_CoreWebView2Ready(sender As Object, e As EventArgs) Handles WebView.CoreWebView2Ready
        Dim URL As New Uri("https://bing.com")
        WebView.Source = URL
    End Sub
```

**Comment by ukandrewc** [comment]
I suspect @mikemeinz is on the right lines with initialization, I can open three or more instances with multiple tabs in each.
![sample](https://user-images.githubusercontent.com/30246320/89577193-39e7b780-d828-11ea-9346-547adacccd9f.png)

**Comment by champnic** [comment]
Thanks for the added info - sounds like explicitly calling EnsureCoreWebView2Async may be a workaround, but we don't mean for it to be required in this scenario. Still a bug on our end to make sure just setting WV2.Source works properly for initialization in multiple instances.

@Merko do you want to try @mikemeinz's suggestion in the meantime?

**Comment by ukandrewc** [comment]
I am using EnsureCoreWebView2Async in my code, so that probably explains it.

**Comment by ukandrewc** [comment]
> I also can't use Stable version of WebView2, only Pre-release (if I choose any of the other, it won't appear in the component list)

From [Release Notes](https://docs.microsoft.com/en-us/microsoft-edge/webview2/releasenotes): While WebView2 is a preview, the .NET APIs are in the pre-release package.

**Comment by oggy22** [comment]
I reproed it on last few versions including 0.9.579-prerelease, but this doesn't repro on the latest which will be published soon, so I am closing this bug. Please reopen it if it still reproes for you once we publish the latest package.

**Comment by champnic** [comment]
This should be fixed in [WebView2 SDK 1.0.674-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.674-prerelease). If this is still failing please let us know. Thanks!
